src/utilities/util_dump.c

```diff
@@ -80,8 +83,9 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
     key = NULL;
     checkpoint = ofile = simpleuri = uri = timestamp = NULL;
     hex = json = pretty = reverse = search_near = false;
+    window = 0;
 
-    while ((ch = __wt_getopt(progname, argc, argv, "c:f:k:t:jnprx?")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "c:f:k:t:jnprwx?")) != EOF)
```

Comment:
Once `w` takes an argument, you will need to have the following:
```suggestion
    while ((ch = __wt_getopt(progname, argc, argv, "c:f:k:t:w:jnprx?")) != EOF)
```

Reply:
added